ambry-rest/src/main/java/com/github/ambry/rest/PublicAccessLogHandler.java

```diff
@@ -116,7 +123,11 @@ public void write(ChannelHandlerContext ctx, Object msg, ChannelPromise promise)
       }
       if (shouldReset) {
         logDurations();
-        publicAccessLogger.logInfo(logMessage.toString());
+
+        if(publicAccessLogger.structuredLoggingEnabled()) {
+          publicAccessLogger.logInfo(structuredLogMessage);
+        }
+        else publicAccessLogger.logInfo(logMessage.toString());
```

Comment:
nit: formatting issue. This should be

```
} else {
    publicAccessLogger.logInfo(logMessage.toString());
}
```

Reply:
Yeah. You could probably use '⌘⌥L' option to format file

Reply:
I ran the IntelliJ command suggested by Arun on this and the other changed files.